be/test/exec/vectorized/csv_scanner_test.cpp

```diff
@@ -35,7 +35,8 @@ class CSVScannerTest : public ::testing::Test {
         tuple_desc_builder.build(&desc_tbl_builder);
 
         DescriptorTbl* desc_tbl = nullptr;
-        Status st = DescriptorTbl::create(&_obj_pool, desc_tbl_builder.desc_tbl(), &desc_tbl, config::vector_chunk_size);
+        Status st =
```

Comment:
It seems that the line length limit of your clang-format is shorter than ours. We adopt 120 line length, not 100 any more.
you can change to 120 and reformat.